File: C#/Leetcode/DataStructures/TrieNode.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LeetcodeSolutions.DataStructures
{
    public class TrieNode
    {
        private static int AlphabetSize = 26;

        public TrieNode[] Children;

        public bool IsEndOfWord = false;

        public TrieNode()
        {
            Children = new TrieNode[AlphabetSize];

            for (int i = 0; i < AlphabetSize; i++)
                Children[i] = null;
        }

        public TrieNode(int alphabetSize) : this()
        {
            AlphabetSize = alphabetSize;
        }
    }
}

File: Java/DataStructures/TrieNode.java
package DataStructures;

public class TrieNode {
  public boolean isEndOfWord;
  public TrieNode[] children;

  public TrieNode() { }

  TrieNode(boolean isEndOfWord, TrieNode[] children) {
    this.isEndOfWord = isEndOfWord;
    this.children = children;
  }
}
